alarms/Telegram/telegram_alarm.py

```diff
@@ -64,10 +70,28 @@ def set_alert(self, settings, default):
 		alert['title'] = settings.get('title', default['title'])
 		alert['body'] = settings.get('body', default['body'])
 		alert['location'] = parse_boolean(settings.get('location', default['location']))
+		alert['sticker_id'] = settings.get('send_stickers', default['sticker_id'])
 		return alert
  		
 	#Send Alert to Telegram
  	def send_alert(self, alert, info):
```

Comment:
I don't understand this 'sticker' parameter. Doesn't it need to be self.sticker or something to work in the class/ object thing? Also where is it meant to go? Inside send_alert or at the end of set_alert?

I tried adding the other suggestions but it doesn't work yet so I'm struggling to work out what is not correct.